frontend/src/toolbar/flags/featureFlagsLogic.ts

```diff
@@ -97,6 +105,12 @@ export const featureFlagsLogic = kea<featureFlagsLogicType>({
             ;(window['posthog'] as PostHog).onFeatureFlags((_, variants) => {
                 toolbarLogic.actions.updateFeatureFlags(variants)
             })
+            const locallyOverrideFeatureFlags = (window['posthog'] as PostHog).feature_flags.instance.get_property(
```

Comment:
I feel like we should expose this property from `posthog-js` through something like feature_flags.hasLocalOverrides(). I couldn't get my dev env for `posthog-js` running though...